generators/test/templates/template_test.go

```diff
@@ -0,0 +1,57 @@
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/gruntwork-io/terratest/modules/test-structure"
+)
+
+func TestTerraformSshExample(t *testing.T) {
+	t.Parallel()
```

Comment:
What's the purpose to have t.Parallel() here?

Reply:
Good question. Terratest official examples usually use `t.Parallel()` in their code but I think it's not a big issue.

Reply:
If you don't have multiple cases to run in parallel, please don't put it.

---
In reply to: [208439972](https://github.com/Azure/generator-az-terra-module/pull/2#discussion_r208439972) [](ancestors = 208439972)